app/src/main/java/pm/gnosis/heimdall/ui/dialogs/share/BaseShareAddressDialog.kt

```diff
@@ -20,22 +20,20 @@ import java.math.BigInteger
 
 abstract class BaseShareAddressDialog : BaseShareQrCodeDialog() {
```

Comment:
Minor thing but according to the style guide `open protected fun onCopiedToClipboard() {}` should be `protected open fun onCopiedToClipboard() {}`